scheduler/src/cook/mesos/api.clj

```diff
@@ -265,7 +265,7 @@
       (merge {:framework-id (s/maybe s/Str)
               :status s/Str
               :state s/Str
-              :submit-time PosInt
+              :submit-time (s/maybe PosInt)
```

Comment:
There's a comment:

; due to a bug, submit time may not exist for some jobs

In that case, as it stands you can't query those jobs. To me, I'd rather update the schema. We can also populate a default value instead.